### PR TITLE
Add cache warm example

### PR DESCRIPTION
**Pull request by hex337**

### CODE REVIEW

**Comment by hex337** [comment]
@josephwegner Nothing really crazy in this recipe, but if you have any thoughts, lemme know.

**Comment by josephwegner** [comment]
I'm gonna be picky.

Can you create a constant at the top called `MAX_AGE`, and then calculate the period (ie: `every (MAX_AGE/2).hours).. and use the constant in the `keen` step.

File: cache-warmer/cache_warmer_job.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+require 'pushpop'
+require 'pushpop-keen'
+
+job 'Warm up keen cached queries' do
+
+  ever 4.hours
```
Comment (josephwegner) [inline cache-warmer/cache_warmer_job.rb]: every?

**Comment by hex337** [comment]
Using a constant is tricky. If someone wants the max age to be something that isn't hours, then the `max_age/whatever` won't work for the every interval.

I'm torn between making it as flexible as possible versus just static and assume most people will be able to modify as needed.

**Comment by hex337** [comment]
I guess max_age could just always be defined in seconds.

**Comment by josephwegner** [comment]
`MAX_AGE = 4.hours` will return seconds. :open_mouth: :boom:

**Comment by josephwegner** [comment]
those emoji are the equivalent of :mindblown: FWIW

**Pull request merged**